public/components/overview/office-panel/config/drilldown-rules-config.tsx

```diff
@@ -70,6 +70,9 @@ export const drilldownRulesConfig = {
                     { field: 'data.office365.UserId', label: 'User ID' },
                     { field: 'data.office365.ClientIP', label: 'Client IP' },
                     { field: 'data.office365.Operation', label: 'Operation' },
+                    { field: 'rule.description', label: 'Description' },
+                    { field: 'rule.level', label: 'Level' },
+                    { field: 'rule.id', label: 'Rule ID' },
```

Comment:
Rule information was deliberately omitted from this specific drilldown as it is a Rule Description Drilldown, every element in this table will share the same `Description`, `Level` and `Rule ID`.
It could be included in the table but ideally it would be included in the title of the drilldown, perhaps on a future iteration.

Reply:
I think I will add for consistency. Maybe we could discuss that in the next iterations.